apps/damap-frontend/src/app/components/layout/layout.component.css

```diff
@@ -7,9 +7,13 @@
   overflow: visible;
 }
 
+.mat-sidenav {
+  width: 301px;
+}
+
```

Comment:
do we want to set this to an absolute value?

Reply:
Yes, otherwise it scales with screen-width and it looks really bulky on bigger screens - at least thats my opinion.